flake.nix

```diff
@@ -107,8 +106,7 @@
           # tools we want in our shell, from hackage
           shell.tools =
             {
-              # for now we're using latest cabal for `cabal haddock-project` fixes
-              # cabal = "3.10.3.0";
+              cabal = "3.14.1.1";
```

Comment:
* [ ] TODO bump in GHA if nix is happy with that version

Reply:
@carbolymer> why this specific version?

Reply:
It's just the latest, to not have to build cabal ourselves from a specific commit. That was done to include a fix for haddocks building.